### DISCUSSION TITLE
如何在Better “Note”中更改插入文献引注的格式？我指的是文中的引用格式，不是文末的参考文献列表。

### DISCUSSION THREAD

**Discussion by iampearl918** [windingwind/zotero-better-notes#1119] [Q&A]
请问better note 笔记中，如何更改引用文献的样式呢？首次插入时跳出来选择样式的窗口，之后就不再有了，想更改样式找不到地方！ 
备注：不是word中更改样式，是zotero的note中。我现在直接用note写文章了，感觉markdown语法写作更方便。

只是在我第一次”插入引注“时弹出来窗口让我选择样式，之后再也无法修改了！哪怕不在同一个better “note”中，甚至我把之前用的样式删除了，Better “note”中还是原样式的格式🧐就默认这一个，再也改不了了呢😭

提示：不是指添加注释时自动生成的那个注释的引用格式，是自己插入的文献引用。Better Note最新版，在笔记编辑窗口的最上方一排工具按钮中，那个“插入引注“。

**Comment by ChartreuseBurbon** [comment]
没想到有人在同一天跟我产生了同样的问题，蹲解答

**Comment by iampearl918** [comment]
🤝甚至我昨天试了个笨办法，把之前用的样式删除了，它还是原样式的格式🧐这就很奇特

**Comment by windingwind** [comment]
笔记中的格式由Zotero确定，目前无法自定义。导出markdown的格式可以自定义

**Reply by iampearl918** [reply]
咦，但是我第一次在笔记中“插入引注”时，弹出来选择样式的窗口、跟word一样的窗口，我还选了个第三方自编的样式。
所以我才以为可以调整样式，只是不知道怎么调整

**Comment by LeviHonestus** [comment]
请问解决了吗，在用note写汇总，想改参考文献格式
